src/collections/_documentation/platforms/javascript/sourcemaps.md

```diff
@@ -404,21 +404,25 @@ var fs        = require('fs'),
     path      = require('path'),
     sourceMap = require('source-map');
 
-// file output by Webpack, Uglify, etc.
-var GENERATED_FILE = path.join('.', 'app.min.js.map');
+async function run() {
+    // file output by Webpack, Uglify, etc.
+    var GENERATED_FILE = path.join('.', 'app.min.js.map');
 
-// line and column located in your generated file (e.g. source of your error
-// from your minified file)
-var GENERATED_LINE_AND_COLUMN = {line: 1, column: 1000};
+    // line and column located in your generated file (e.g. source of your error
+    // from your minified file)
+    var GENERATED_LINE_AND_COLUMN = {line: 1, column: 1000};
 
-var rawSourceMap = fs.readFileSync(GENERATED_FILE).toString();
-var smc = new sourceMap.SourceMapConsumer(rawSourceMap);
+    var rawSourceMap = fs.readFileSync(GENERATED_FILE).toString();
+    var smc = await new sourceMap.SourceMapConsumer(rawSourceMap);
```

Comment:
@rchl can you make this example use promises instead of top-level await?

Reply:
I can do that but technically speaking, this is not a top-level await. It's inside a function.

Reply:
@rchl Ah you're right 👍 I've meant if the example can be conveyed without this wrapping function.

Reply:
It can but IMO using `then` form of promise is less readable. Are you sure you want me to change that?